pkgs/os-specific/linux/pcimem/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "pcimem";
+  version = "2018-08-29";
+  commit = "09724edb1783a98da2b7ae53c5aaa87493aabc9b";
+
+  src = fetchurl {
+    urls = [
+      "https://github.com/billfarrow/pcimem/archive/${commit}.tar.gz"
+    ];
+    sha256 = "0p7gxg3kqx3zsykmixy9zla8yvf3x8qdm0s7rcs6anvzcxpsi3rb";
+  };
+
+  outputs = [ "out" "doc" ];
+
+  makeFlags = [ "CFLAGS=-Wno-maybe-uninitialized" ];
+
+  installPhase = ''
+    install -D pcimem "$out/bin/pcimem"
+    install -D README "$out/doc/README"
+  '';
+
+  meta = with stdenv.lib; {
+    description = ''a simple method of reading and writing to memory registers on a PCI card'';
+    homepage = "https://github.com/billfarrow/pcimem";
+    license = licenses.gpl2Plus;
```

The repo does not mention it is a GPLv2+ code

```suggestion
    license = licenses.gpl2Only;
```